### PR TITLE
README.md rework

### PR DESCRIPTION
**Pull request by Happychap612**

### CODE REVIEW

**Comment by Happychap612** [comment]
@sdomino Better?

File: README.md [hunk 1]
```diff
@@ -1,58 +1,30 @@
...
 
 # Rails from scratch
-
-Run a Rails app locally, install nothing besides nanobox. 
+Run a local rails app with nothing but Nanobox.
 
 <a href="https://nanobox.io/download"><img src="https://guides.nanobox.io/assets/quickstart-icons/download.png" /></a>
 
+# Installation and Setup
+1. Clone the nanobox repo
```
Comment (sdomino) [inline README.md]: Clone the nanobox repo -> Clone this repo

File: README.md [hunk 2]
```diff
@@ -1,58 +1,30 @@
...
+      `git clone https://github.com/nanobox-quickstarts/nanobox-rails.git`
+2. Don't forget to cd into the new directory
```
Comment (sdomino) [inline README.md]: Don't forget to cd into the new directory -> Change into the directory

File: README.md [hunk 3]
```diff
@@ -1,58 +1,30 @@
...
+        `cd nanobox-rails`
+3. Add a local DNS record
+        `nanobox dns add local rails.dev`
+4. Run rake to create a database
```
Comment (sdomino) [inline README.md]: Run rake to create a database -> Create the database

**Review by sdomino — CHANGES_REQUESTED** [review]
Looks good. I made a few minor suggestions. Once those are updated I'll merge. Thanks!

**Comment by Happychap612** [comment]
@sdomino There we go. Let me know if theres anything else. I'll try and respond faster this time 😄

**Pull request merged**